### DISCUSSION TITLE
The code generator has deoptimised the styling since it exceeds the max of 500KB.

### DISCUSSION THREAD

**Discussion by satvik-1203** [babel/babel#13676] [Q&A] [answered]
> The code generator has deoptimized the styling of /Users/**/Desktop/portfolio/unknown as it exceeds the max of 500KB.

So I have been getting this error since I started using heavy SVGs, there is no clear answer in the internet. Is there any plugin or should I ignore that SVG with babel ignore? 

**SVG size:** 3MB

**Comment by nicolo-ribaudo** [comment, marked as answer]
It's just a warning (not an error) that can be safely ignored.

However, if you want to silence it, you might try to set the https://babeljs.io/docs/en/options#compact option to `true`. (I'm not 100% sure that it works, but that's effectively what the warning is doing).

Btw, why are you compiling SVGs with Babel?

**Reply by satvik-1203** [reply]
I think since I'm importing the SVG in my TSX file, it's trying to babel that as well or something, I'm not entirely sure. it's not an error I see when  I remove that import. Okay, so it's not something I should worry about tho right?

**Reply by nicolo-ribaudo** [reply]
Yeah, it won't cause any problem. That warning is just telling you "if you are trying to read the Babel-generated code but it's minified/compressed, this is why".

**Comment by Mohsinsiddi** [comment]
We can ignore it!

**Reply by iamjuangonzalez** [reply]
Como lo ignoramos? No entiendo en que parte debo ignorarlo.

**Reply by Mezeobidi** [reply]
It means that you should not pay attention to it

**Comment by AlastairTaft** [comment]
Is there a way to force it to make readable code? I'm not worried about the compile time.

EDIT: [This](https://babeljs.io/docs/options#compact) is the option you need.
